### ISSUE TITLE
icons overlap text in menu (on HiDPI screen)

### ISSUE THREAD

**Issue by Rabijns** [spyder-ide/spyder#11208]
### Issue Report Checklist
*  [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [ ] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [ ] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [ ] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
in the menus, the icons overlap with the text of the menu item
![image](https://user-images.githubusercontent.com/9770955/71645302-9a34da80-2cd6-11ea-9899-9b176d99599e.png)

Computer is a HP laptop with HiDPI screen (3840 x 2160). The settings in the OS (linux Mint 19.2) are set to "user interface scaling : Double (Hi-DPI).

### What steps reproduce the problem?

1. open spyder
2. open menu
issue occurred on a fresh install (via anaconda) of spyder4

### What is the expected output? What do you see instead?
Icon should not overlap with text

### Paste Traceback/Error Below (if applicable)

```python-traceback

PASTE TRACEBACK HERE

```

## Versions

* Spyder version: 4.0.0
* Python 3.7.5 64-bit 
* Qt 5.9.6
* PyQt5 5.9.2
* Linux 4.15.0-72-generic 

### Dependencies

```
cloudpickle >=0.5.0          :  1.2.2 (OK)
pygments >=2.0               :  2.5.2 (OK)
qtconsole >=4.6.0            :  4.6.0 (OK)
nbconvert >=4.0              :  5.6.1 (OK)
sphinx >=0.6.6               :  2.3.0 (OK)
pylint >=0.25                :  2.4.4 (OK)
psutil >=0.3                 :  5.6.7 (OK)
qtawesome >=0.5.7            :  0.6.0 (OK)
qtpy >=1.5.0                 :  1.9.0 (OK)
pickleshare >=0.4            :  0.7.5 (OK)
zmq >=17                     :  18.1.0 (OK)
chardet >=2.0.0              :  3.0.4 (OK)
numpydoc >=0.6.0             :  0.9.1 (OK)
spyder_kernels >=1.8.1;<2.0.0:  1.8.1 (OK)
qdarkstyle >=2.7             :  2.7 (OK)
atomicwrites >=1.2.0         :  1.3.0 (OK)
diff_match_patch >=20181111  :  20181111 (OK)
intervaltree                 :  None (OK)
watchdog                     :  None (OK)
keyring                      :  None (OK)
pexpect >=4.4.0              :  4.7.0 (OK)
pympler                      :  None (OK)
sympy >=0.7.3                :  1.5 (OK)
cython >=0.21                :  0.29.14 (OK)
IPython >=4.0                :  7.10.2 (OK)
matplotlib >=2.0.0           :  3.1.1 (OK)
pandas >=0.13.1              :  0.25.3 (OK)
numpy >=1.7                  :  1.17.4 (OK)
scipy >=0.17.0               :  1.3.2 (OK)
pyls >=0.31.2;<0.32.0        :  0.31.2 (OK)
xdg >=0.26                   :  0.26 (OK)
rtree >=0.8.3                :  0.8.3 (OK)
```

**Comment by Rabijns** [comment]
update:
I went into preferences and selected 'enable auto high dpi scaling'. The overlap is no longer there.
I leave it up to you to decide if this is therefore a user error or a bug, but for me, the item can be closed.
Thanks for all the hard work on Spyder V4.

**Comment by ccordoba12** [comment]
> The overlap is no longer there. I leave it up to you to decide if this is therefore a user error or a bug

You have to activate that option for things to work as expected in HiDPI screens.

**Comment by ccordoba12** [comment]
> Thanks for all the hard work on Spyder V4.

No problem. Thanks for your kind words!

**Comment by lindsayfowler** [comment]
Icons mingling with menu text is still a problem in Spyder 5, and enabling auto high DPI scaling in Preferences > Application doesn't seem to fix it.

**Comment by Tristhal** [comment]
I have the same issue. Prior to updating to 5.0.0 the icon's had no issues and I checked. Auto high DPI scaling is enabled. I also disabled and re-enabled it.
![image](https://user-images.githubusercontent.com/10468841/115948616-25724a80-a49d-11eb-963c-ee879296f5e7.png)

**Comment by lindsayfowler** [comment]
So what is preferred: start a new issue, even though this is an old issue?

**Comment by lukaswrm** [comment]
Same for Ubuntu 20.04

**Comment by ccordoba12** [comment]
This is already fixed and part of our 5.0.1 version, released a week and a half ago and soon to be part of Anaconda.
